[Korean] Update vue-router 2.5

ChangJoo-Park opened the pull request.

ChangJoo-Park committed:
[Korean] Update vue-router 2.5

kazupon commented on docs/kr/advanced/navigation-guards.md:
```diff
@@ -32,8 +32,20 @@ router.beforeEach((to, from, next) => {
+
+> 2.5.0에서 추가됨
+
+In 2.5.0+ you can register a global guard with `router.onResolve`. This is similar to `router.beforeEach`, with the difference that resolve guards will be called right before the navigation is confirmed, **after all in-component guards and async route components are resolved**.
```
Did you forget to remove it?

ChangJoo-Park replied:
Oh, It is my mistake 😭

kazupon reviewed (changes requested):
Thank you for your PR!
Please check my review comment 💬

ChangJoo-Park committed:
[Korean] remove english Global Resolve Guards

Global Resolve Guards and Global After Hooks headings are not need to translate in Korean. It is make sense in English than Korean

ChangJoo-Park commented:
Sorry @kazupon :sob: I forgot to remove original sentences

kazupon commented:
Thanks!

ChangJoo-Park commented:
:)
